src/pages/AddSongForm.js

```diff
@@ -60,6 +61,10 @@ function AddSongForm(props) {
                     <td class="button_alignment">
                         <input type="button" value="Add Song" onClick={() => props.handleAdd(row)}/>
                     </td>
+                    <td class="button_alignment">
+                        <Link to={"/Songs/"+row.name+"/"+row.artists[0].name+"/"+row.album.name}><button type="button"
+                    class="button_playlist">INFO</button></Link>
+                </td>

```

Comment:
So cool to see how our project has progressed! I'm glad you found such a simple way to use the API to our advantage

Reply:
I am glad as well that we managed to use the Spotify API